ios/PurchasesHybridCommon/PurchasesHybridCommon/Package+HybridAdditions.swift

```diff
@@ -9,6 +9,21 @@
 import Foundation
 import RevenueCat
 
+public extension PresentedOfferingContext {
+
+    var dictionary: [String: Any] {
+        let dict: [String: Any?] = [
+            "offeringIdentifier": self.offeringIdentifier,
+            "placementIdentifier": self.placementIdentifier,
+            "targetingRevision": self.targetingContext?.revision,
+            "targetingRuleId": self.targetingContext?.ruleId,
+        ]
+
+        return dict.compactMapValues { $0 }
```

Comment:
Since we are using `compactMapValues`, that means we won't have keys with null values right? My main concerns are:
- It's a slight difference in behavior from Android where we can have these keys with null values
- In the typescript code, we've set the types to `PresentedOfferingContext | null`, but do not support `undefined` which I think would be what would happen if we strip null key/values... 
I think we should either not compact the values here or fix it in android and typescript

Reply:
Ahhhhh, good point good point! I'll remove the compact map 👍